build.gradle

```diff
@@ -20,8 +20,14 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
-        maven { url 'https://maven.google.com' }
-        google()
+        maven {
+            url 'https://maven.google.com/'
+            name 'Google'
+        }
+        maven {
+            url "https://jitpack.io"
+//            dont use proxy when syncing it
```

Comment:
No place for a comment

Reply:
Just a heads up for the person taking over after me

Reply:
Not useful at all, since the next person would probably be working in a completely different environment, and this will only add to the confusion if something isn't working. On a side note, why are you using a proxy at all?